assets/src/js/homepage_transparent_nav.js

```diff
@@ -0,0 +1,45 @@
+export const setupTransparentNavHomepage = () => {
+  let isScrolled = false;
+  ready(() => {
+    if (!onHomePage()) {return;}
```

Comment:
For better readability and consistency across our codebase, it would be better to move this `return;` statement to the next line

```
if (!onHomePage()) {
  return;
}

```

Reply:
I'm also not sure we need this `onHomePage` function since it's only used once and is only one line of code?

Reply:
Ideally I would also declare the needed functions (such as `ready` before using them, not after